ui/apps/platform/src/Containers/SystemHealth/DashboardPage.js

```diff
@@ -27,6 +28,9 @@ const SystemHealthDashboardPage = () => {
     const hasReadAccessForCluster = hasReadAccess('Cluster');
     const hasReadAccessForIntegration = hasReadAccess('Integration');
 
+    const { isFeatureFlagEnabled } = useFeatureFlags();
+    const hasAccessForScannerV4 = isFeatureFlagEnabled('ROX_SCANNER_V4');
```

Comment:
nit: the name of this variable may be misleading (unless this is a UI convention).  The feature flag is unrelated to 'access'.  Perhaps `isScannerV4Enabled` would be more accurate?